standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestTablesCreateDropAlterTruncate.java

```diff
@@ -1595,7 +1591,7 @@ private Table getTableWithAllParametersSet() throws MetaException {
                .setNumBuckets(4)
                .setRetention(30000)
                .setRewriteEnabled(true)
-               .setType("VIEW")
+               .setType("VIRTUAL_VIEW")
```

Comment:
Looks Interesting. Good catch! We don't actually have the `VIEW` type. But i saw it in multiple places. `TestTablesGetExists` also has same misuse, can we fix it together? @deniskuzZ WDYT?
```suggestion
               .setType(TableType.EXTERNAL_TABLE.name())
```